src/feature/client/entrynodes.h

```diff
@@ -515,15 +527,15 @@ MOCK_DECL(STATIC circuit_guard_state_t *,
 STATIC entry_guard_t *entry_guard_add_to_sample(guard_selection_t *gs,
                                                 const node_t *node);
 STATIC entry_guard_t *entry_guards_expand_sample(guard_selection_t *gs);
-STATIC char *entry_guard_encode_for_state(entry_guard_t *guard);
+STATIC char *entry_guard_encode_for_state(entry_guard_t *guard, int i);
```

Comment:
Looks like a typo or an old variable name
```suggestion
STATIC char *entry_guard_encode_for_state(entry_guard_t *guard, int dense_sampled_index);
```